zebra-network/src/peer_set/inventory_registry.rs

```diff
@@ -146,11 +145,7 @@ impl<T: Clone> InventoryStatus<T> {
 impl InventoryRegistry {
     /// Returns a new Inventory Registry for `inv_stream`.
     pub fn new(inv_stream: broadcast::Receiver<InventoryChange>) -> Self {
-        let interval = Duration::from_secs(
-            POST_BLOSSOM_POW_TARGET_SPACING
-                .try_into()
-                .expect("non-negative"),
-        );
+        let interval = INVENTORY_ROTATION_INTERVAL;
```

Comment:
This makes us re-check peers for new blocks slightly faster.

We just added a lot of tracking for missing blocks, so it's safe to retry a bit faster.